_build/data/docs/commandline.rst

```diff
@@ -3,18 +3,45 @@ Command Line Options
 
 The EventGhost main executable accepts the following command line arguments: 
 
+.. cmdoption:: -configdir <directory>
+
+    Instructs EventGhost to use the directory <directory> to store and
+    retrieve its settings. Without this option EventGhost uses a directory in
+    the application data folder of your machine for storing its settings.
+    For example, through this option you can change the folder to a location
+    on a USB stick to make EventGhost portable.
+
```

Comment:
I would remove the comments about being portable. because EG technically does not conform to being a portable application because of its creation/use of ProgramData\EventGhost. and it does not remove the folder after being run. so it leaves behind data. and a portable program will not/should not do this.